viewCategory.php

```diff
@@ -0,0 +1,30 @@
+<?php
```

Comment:
I can't comment on the file itself, but `trees.JPG` above is a pretty big file (425 KB). I doubt it needs to be that big, and your page will load faster if you scale it down to a smaller size.